stock_location_orderpoint_cleanup/wizards/stock_location_orderpoint_cleanup.xml

```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<!-- Copyright 2023 ACSONE SA/NV
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+
+    <record model="ir.ui.view" id="stock_location_orderpoint_cleanup_form_view">
+        <field
+            name="name"
+        >stock.location.orderpoint.cleanup.form (in stock_location_orderpoint_cleanup)</field>
+        <field name="model">stock.location.orderpoint.cleanup</field>
+        <field name="arch" type="xml">
+            <form string="Stock Location Orderpoint Cleanup">
+                <!-- TODO -->
+                <p class="oe_grey">
+                    This will launch the cleanup process for all selected location orderpoints.<br
+                    />
+                    Check the "Run After" box if you want to run the orderpoints after
+                    that cleanup.
+                </p>
+                <group>
+                    <field name="run_after" />
+                    <field name="orderpoint_ids" />
+                </group>
+                <footer>
+                    <button
+                        name="doit"
+                        string="Cleanup"
+                        class="btn-primary"
+                        type="object"
+                    />
+                    <button string="Cancel" class="btn-default" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+
+    <record
+        model="ir.actions.act_window"
+        id="stock_location_orderpoint_cleanup_act_window"
+    >
+        <field name="name">Stock Location Orderpoint Cleanup</field> <!-- TODO -->
+        <field name="res_model">stock.location.orderpoint.cleanup</field>
+        <field name="view_mode">form</field>
+        <field name="context">{}</field>
+        <field name="target">new</field>
+    </record>
```

Comment:
Wouldn't it be not nice to select a bunch of orderpoints and open this wizard for more then just one orderpoint?
Right now it is only possible to do it via the button shown in the form view.